OpenRA.Mods.D2k/Warheads/DamagesConcreteWarhead.cs

```diff
@@ -0,0 +1,39 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using OpenRA.Mods.Common.Warheads;
+using OpenRA.Mods.D2k.Traits;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.D2k.Warheads
+{
+	[Desc ("Interacts with the BuildableTerrainLayer trait.")]
+	public class DamagesConcreteWarhead : Warhead
+	{
+		[Desc("The rock tile ID to fall back to.")]
+		public readonly ushort Template = 266;
```

Comment:
The last revision hid the previous comment, so repeating here: removing the tile needs to reset the TerrainLayer to its default state of rendering nothing.  Drawing a duplicate tile is bogus.

Reply:
Fixed that as well. Now it preserves the smudges.